aicssegmentation/workflow/workflow.py

```diff
@@ -153,6 +153,35 @@ def execute_all(self) -> np.ndarray:
             self.execute_next()
         return self.get_most_recent_result()
 
+    def run_step(self, i: int, parameters: Dict[str, Any] = None) -> np.ndarray:
+        """
+
+        Args:
+            i: step number (0 indexed) that you want to run
+
+        Returns:
+
```

Comment:
Missing return value description here, also would be nice to have a description in the doc block about what `run_step` is intended to do, but otherwise looks good to me!